pkgs/development/node-packages/node-packages.nix

```diff
@@ -89724,7 +89724,7 @@ in
     buildInputs = globalBuildInputs;
     meta = {
       description = "Production process manager for Node.JS applications with a built-in load balancer.";
-      homepage = http://pm2.keymetrics.io/;
+      homepage = https://pm2.keymetrics.io/;
```

Comment:
```suggestion
      homepage = "https://pm2.keymetrics.io/";
```

Reply:
I force pushed this suggestion and resolved the merge conflicts to get this merged.